client/containers/Pub/PubDocument/PubBody.js

```diff
@@ -115,6 +117,15 @@ const PubBody = (props) => {
 						updateLocalData('collab', { editorChangeObject: editorChangeObject });
 					}
 				}}
+				onError={(err) => {
+					setEditorError(err);
+					if (window.sentryIsActive) {
```

Comment:
I hope this never happens in practice, but if this callback were triggered during a server-side render, I believe it would crash the request with a 500 because `window` is undefined. We probably want `typeof window !== 'undefined' && window.sentryIsActive` here.

Reply:
I thought about that and figured any Editor errors would've been caught when calling `getBranchDoc` to build the doc from Firebase steps in the first place. But, better to add the extra safety check. Will add a commit in a moment.